src/platform/user/profile/vet360/reducers/index.js

```diff
@@ -112,7 +112,7 @@ export default function vet360(state = initialState, action) {
           [action.fieldName]: {
             ...state.fieldTransactionMap[action.fieldName],
             isPending: false,
-            transactionId: action.transaction.data.attributes.transactionId,
+            transactionId: action.transaction?.data?.attributes?.transactionId,
```

Comment:
Was this something you wanted to commit? I wonder if we actually do want to blow up in this case since the `action.transaction` _should_ be properly formed if it reaches this point?